src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BeeListener.java

```diff
@@ -0,0 +1,51 @@
+package io.github.thebusybiscuit.slimefun4.implementation.listeners;
+
+import java.util.Optional;
+
+import org.bukkit.entity.Bee;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.cscorelib2.inventory.ItemUtils;
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.core.attributes.ProtectionType;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+
+/**
+ * The listener for Hazmat Suit's {@link Bee} sting protection.
+ * Only applied if the whole set is worn.
+ *
+ * @author Linox
+ *
+ */
+public class BeeListener implements Listener {
+
+    public BeeListener(SlimefunPlugin plugin) {
+        plugin.getServer().getPluginManager().registerEvents(this, plugin);
+    }
+
+    @EventHandler
+    public void onDamage(EntityDamageByEntityEvent e) {
+        if (e.getDamager() instanceof Bee && e.getEntity() instanceof Player) {
+
+            Player p = (Player) e.getEntity();
+            Optional<PlayerProfile> optional = PlayerProfile.find(p);
+            if (!optional.isPresent()) {
+                PlayerProfile.request(p);
+                return;
+            }
+
+            PlayerProfile profile = optional.get();
+            if (profile.isProtected(ProtectionType.BEES)) {
+                for (ItemStack armor : p.getInventory().getArmorContents()) {
+                    ItemUtils.damageItem(armor, 1, false);
+                }
+                e.setDamage(0D);
```

Comment:
Should cancel. This will mess with other plugins.

Reply:
Cannot when cancelled bee doesn't stop from trying to attack you.

Reply:
Hmmm, what about setting the anger level or target or such?

Reply:
That would cause more issues most likely.